### PR TITLE
update ssopodcount alert to check for at least 4 pods to allow for scaling 

### PR DESCRIPTION
**Pull request by ciaranRoche**
JIRA -> https://issues.redhat.com/browse/INTLY-8600

## Verification Steps
As the verifier of the PR the following process should be done:
After installing RHMI :  
- Scale pods in the `sso` namespace up
- Verify `SSOPodCount` alert **does not** fire
- Scale pods in the `sso` namespace down
- Verify `SSOPodCount` alert **does** fire

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 

## Checklist

- [ ] Updated [Changelog](https://github.com/integr8ly/installation/blob/master/CHANGELOG.md)
- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

Is an upgrade task required and are there additional steps needed to test this?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by ciaranRoche** [comment]
@david-martin sorry only seeing your comment, fully agree, we should be more specific in terms of pod, happy to get those changes in tomorrow, no need to push over on to the monitoring backlog, unless something more urgent comes in tomorrow.

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Some of these changes may include:
 * [INTLY-8459] - Fix 3Scale probe alerts 
 * [INTLY-8601] - Add dummy/null reciever for UnifiedPushJavaHeapThresholdExceeded alert
 
+* [INTLY-8600] - Updated SSOPodCount alert to check for at least 4 pods to allow for scaling of pods
```
Comment (david-martin) [inline CHANGELOG.md:28]: @ciaranRoche looks like a small change needed to the changelog now

**Pull request merged**